rbac/management/group/definer.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Handler for system defined group."""
+import logging
+
+from management.group.model import Group
+from management.policy.model import Policy
+from management.principal.model import Principal
+from management.role.model import Role
+from tenant_schemas.utils import tenant_context
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def _create_group(tenant, role_list):
+    """Create the group object in the database."""
+    with tenant_context(tenant):
+        name = "Platform group"
+        group_description = "Default group that contains default permissions for new users."
+        policy_description = "Platform default policy that is binded with default group."
+
+        principals = list(Principal.objects.all())
+        group = Group.objects.create(name=name,
```

Comment:
I think we'll just want to create this when it doesn't exist? Seeds are run more than just when a tenant is created.

Reply:
Right, I was thinking separate updating existing tenant into a followup PR. I will add that here then.